frontend/epfl-hero/view.php

```diff
@@ -60,7 +60,7 @@ function epfl_hero_block( $attributes ) {
     <div class="hero-img">
       <figure class="cover">
         <?php
-        if ($media_url) { ?>
+        if (isset($media_url) && ($media_url!==null)) { ?>
```

Comment:
```suggestion
        if (isset($media_url)) { ?>
```
isset has already a null check.